### ISSUE TITLE
Image is not cleared if set to None in configure method

### ISSUE THREAD

**Issue by janezgolob** [TomSchimansky/CustomTkinter#2494]
I am not sure if this is supposed to be supported, but I naively expected that my_label.configure(image=None) would clear the image from the label. 

https://github.com/TomSchimansky/CustomTkinter/blob/b9b57d95c7d3bb821a2260527315b6079161f5c9/customtkinter/windows/widgets/ctk_label.py#L143

**Comment by mate02102003** [comment]
I just checked what is the behavior if you would have done this with the base `tkinter` library. It doesn't does anything, it just keeps the image the same.

**Comment by mate02102003** [comment]
But if you do this:
```python
my_label.configure(image="")
```
It should clear the image.

**Comment by janezgolob** [comment]
Right. I actually ended up with two labels on top of each other, using one of them for text and another one for icons. Just to avoid those warnings.
I am totally fine if you keep the same behavior, though I would argue that what tkinter does is more accidental behavior than intentional. Maybe adding a warning would help already.

**Comment by mate02102003** [comment]
Hi, I looked into the underlying frameworks documentation, the empty string so `""` is the intended way to disable the currently set image.
[tk label image](https://www.tcl.tk/man/tcl9.0/TkCmd/options.html#M-image)
![image](https://github.com/TomSchimansky/CustomTkinter/assets/172080705/1918e0e0-3df7-460f-b483-a10da04227f4)

**Comment by janezgolob** [comment]
OK. Fine. May I suggest removing that warning then?

Although it seems strange at first glance, I agree it is best that customtkinter keeps compatibility with tkinter in this sense as well. Thanks, everyone, for looking into this.

**Comment by FedericoSpada** [comment]
I've hidden the warning if the image is set exactly to "", which is the correct way to delete it.
